pkg/apis/controlplane/v1beta2/types.go

```diff
@@ -88,6 +88,8 @@ type GroupMember struct {
 	Ports []NamedPort `json:"ports,omitempty" protobuf:"bytes,4,rep,name=ports"`
 	// Node maintains the reference to the Node.
 	Node *NodeReference `json:"node,omitempty" protobuf:"bytes,5,opt,name=node"`
+	// Service maintains the reference to the Service.
```

Comment:
maintains -> is

Reply:
Add comments about: it can only be in an AppliedTo Group and can only be a NodePort Service.

Reply:
Done.